src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/MarkMembersAsStaticTests.cs

```diff
@@ -1016,9 +1016,18 @@ public class C : System.Web.HttpApplication
     // The following controller methods can't be made static
     protected void Application_Start() { }
     protected void Application_End() { }
-}");
+}";
 
-            await VerifyVB.VerifyAnalyzerAsync(@"
+            const string editorConfigText = "build_property.UsingMicrosoftNETSdkWeb = true";
```

Comment:
Just to confirm I got things right, we don't expect user to set this in their `.editorconfig` file? This is a msbuild property that is present for web apps, right?

Reply:
Yes, users do not have to do anything.